supertokens_python/recipe/session/framework/django/syncio/__init__.py

```diff
@@ -38,18 +39,21 @@ def verify_session(
     ] = None,
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[[_T], _T]:
-    if user_context is None:
-        user_context = {}
+    _ = user_context
 
     def session_verify(f: _T) -> _T:
         from django.http import HttpRequest
 
         @wraps(f)
         def wrapped_function(request: HttpRequest, *args: Any, **kwargs: Any):
+            nonlocal user_context
             from django.http import JsonResponse
 
             try:
                 baseRequest = DjangoRequest(request)
+                if user_context is None:
+                    user_context = default_user_context(baseRequest)
```

Comment:
this is wrong. We want to modify the user_context even if it's passed by the user to add the default stuff into their passed in user_context.

Reply:
Updated.